openpype/hosts/fusion/deploy/MenuScripts/openpype_menu_launch.py

```diff
@@ -0,0 +1,24 @@
+# Python script to send data using sockets
+
+import socket
+import json
+
+
+def send_message_to_OpenPype(data_to_send):
+    # Serialize the Python dictionary into a JSON string
+    data_json = json.dumps(data_to_send)
+
+    client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    server_address = ("localhost", 12345)
```

Comment:
I'm pretty sure this won't work for multiple fusion sessions open at the same time. They'd both be sending the commands to the same process - and thus, that process that should be "in a certain openpype context, like project, asset, shot, etc." now suddenly has to be in two contexts at the same time. I feel like this will not work and we'll need to have at least a separate process per fusion session.

Also, sessions can crash. I feel there'll be a need to IF no response is received that maybe we'll restart the 'menu' process for the current session.

Reply:
Sounds logical. I was thinking of passing the socket to Fusion over an environment variable. That way it could be randomly picked so it won't already be in use and simply pass it like any other env var.

In short there would be a pre-hook that executes exactly like the old Menu UI object but instead of creating the UI it creates a socket listener and we control the process through socket commands.

Reply:
I think I will need to get some help on how to set up the new process correctly as I have no idea how to do that in OP.

Reply:
Best would be to have one process tho and pass what project and asset is being used. That way we could get tabs in Fusion to work correctly! But that could be done in a later PR